openlibrary/plugins/openlibrary/home.py

```diff
@@ -55,8 +50,9 @@ def get_homepage():
 
 def get_cached_homepage():
     five_minutes = 5 * dateutil.MINUTE_SECS
+    lang = web.ctx.get("lang") or "en"
```

Comment:
We tend to use the default option of the `get` method where possible:

```suggestion
    lang = web.ctx.get("lang", "en")
```